<!-- -*- mode: XML -*- -->
<launch>
    <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
		<param name="destination_frame" value="/base_link"/>
		<param name="cloud_destination_topic" value="/merged_cloud"/>
		<param name="scan_destination_topic" value="/scan_multi"/>
		<!--param name="angle_min" type="double" value="-3.14" /-->
		<!--param name="angle_max" type="double" value="3.14" /-->
    <param name="laserscan_topics" value ="scan scanh" /> <!-- LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
	</node>
</launch>

